index.js

```diff
@@ -271,28 +271,20 @@ module.exports = function({parent}) {
             });
     };
 
-    HttpServerPort.prototype.registerRequestHandler = function(handlers) {
-        if (!(handlers instanceof Array)) {
-            handlers = [handlers];
-        }
-        handlers.forEach(handler => {
-            if (handler.config) {
-                this.log.warn && this.log.warn('Rename "config" to "options" for handler ' + handler.path);
-                // throw new Error('Rename "config" to "options" for handler ' + handler.path);
-            }
-        });
+    HttpServerPort.prototype.registerRequestHandler = function(items) {
+        items.map(handler => (handler.config && this.log.warn && this.log.warn('Rename "config" to "options" for handler ' + handler.path)));
```

Comment:
what if items is not an array? there was a check "if (!(handlers instanceof Array))" which I guess means that there are places where an object is passed when registering a single handler

Reply:
there is a plenty of code that items are array but not checked for it, if there is a error it will be caught on time i think. after all items is plural, when you look @ method fingerprint you will know .